components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -1014,7 +1033,7 @@ - (BOOL)contentScrollsToReveal {
 
 - (CGFloat)topHeaderHeight {
   if (!self.headerViewController) {
-    return 0;
+    return MDCDeviceTopSafeAreaInset();
```

Comment:
One note about this change in particular. I intentionally do not have this behavior modified based on the `shouldUseStickyStatusBar` flag, as it seems to me that this _should_ always be including the safe area inset in the `topHeaderHeight`, as the `contentHeaderHeight` accurately reflects that the height is `0` when there is no `headerViewController` whereas, if I understand correctly, this value should reflect that content height + the necessary safe area offset. I've also updated the accompanying (already existing) unit test to reflect this.

Reply:
So I believe it will cause a _minor_ visual change, but not what I think would be considered a regression. 

This will only effect instances of the drawer where no header is being provided and, if I understand correctly, the only change this will cause in that instance is that it will cause the offset that determines when to animate removing the handle and corner rounding to take into account the top safe area, regardless of whether or not there is a header. 

This seemed to me to be the behavior that should happen, but if that seems off to you, or you would rather for certainty reasons, I'm happy to put this behind the `shouldUseStickyStatusBar` flag as well?

Reply:
An updated thought that struck me about this aspect of this PR: Given that #9471 and #9476 also effect this component, it may be safest to merge this after those two are merged and I have the chance re-base + re-test that there were not any unexpected changes to the behavior here. 

This will also allow me to update this call-site to also use the added `topSafeAreaInset` property rather than `MDCDeviceTopSafeAreaInset`, and verify that the flagged sticky behavior is still as expected. 

If that is the case, do you think it would be possible to get this change in before the next release is cut? Happy to do whatever additional testing you think necessary to get that done as we have a few internal issues that would be helped along with these changes 😄🙏

Reply:
sounds good :) Thanks for thinking this through

Reply:
Im working to get the other changes in before the next release, this one may not make the cut off as the next release is being cut monday morning